src/pages/components/toggletip/usage.mdx

```diff
@@ -0,0 +1,243 @@
+
+</Column>
+</Row>
+
+#### Toggletips verses Tooltips
+
+Toggletips and tooltips are similar visually and both contain a popover and
+interactive trigger element. The two components differ in the way they are
+invoked and dismissed and if the user must interact with the contents. A tooltip
+is exposed on `Hover` or `Focus` when you need to expose brief, supplemental
+information that is not interactive. A toggletip is used on `Click` and `Enter`
+when you need to expose interactive elements, such as button, that a user needs
+to interact with.
+
+The exception here is that definition tooltip can be invoked on either `Hover`
+or `Click` depending on use case.
+
+### When to use
+
+- Use when an interactive element must be placed within the popover
+- Use for quick, in context editing
+- Use for filter panels that overlay content
+
+<Row>
+<Column colLg={8}>
+
+![Overview of when to use toggletips](images/toggle-tip-usage-2.png)
+
+</Column>
+</Row>
+
+### When to not use
+
+Don't use to present critical information or request required input needed to
+complete a workflow. Use the
+[modal component](https://carbondesignsystem.com/components/modal/usage/)
+instead.
+
+## Live demo
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Toggletip anatomy](images/toggle-tip-usage-4.png)
+
+</Column>
+</Row>
+
+1. **UI trigger button:** Usually a button but as long as the trigger is
+   interactive, it can visually change its shape and size depending on the use
+   case.
+2. **Tip:** An indicator that is added to a popover container to help show the
+   relationship between the popover and where it was triggered from. Toggletips can utilize
+   no tip, caret tip, or tab tip types.
+3. **Content area:** Varies based on use case and follows disclosure guidelines.
+4. **Popover container:** Area that contains text and interactive elements.
+
+### Sizing
+
+The width and height of a popover container can vary depending on the amount of
+content placed within it. We recommend to not exceed a popover width size of
+four columns. Refer to the popover component for more specifications.
+
+<Row>
+<Column colLg={8}>
+
+![Toggletip sizing](images/toggle-tip-usage-5.png)
+
+</Column>
+</Row>
+
+### Placement
+
+Toggletips can be used on various part of the UI including, but not limited to,
+headers, data tables, side panels, modals, and read only cards. When active, the
+toggletip popover may be positioned **top, bottom, left,** or **right** to the
+UI trigger.
+
+<Row>
+<Column colLg={8}>
+
+![Toggletip placement](images/toggle-tip-usage-6.png)
+
+</Column>
+</Row>
+
+Placement of a popover in relation to the trigger depends on the type of popover
+used. Visit the popover component for specific guidelines for no tip, caret tip,
+and tab tip types.
+
+## Content
+
+### Main elements
+
+Heading, body, and footer content can vary based on your use case. To see
+examples of content used within toggletips, visit the disclosure pattern.
```

Comment:
I think it could be good to link to the disclose pattern here when we say to visit it. Might lead more people to that page if we give it to them straight away.